index.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="home.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=lato"
+      rel="stylesheet"
+    />
+    <title>First Capstone Project</title>
+  </head>
+  <body>
+    <div class="mobile-menu" id="mobile-menu">
+      <div class="mobile-menu-header">
+        <button type="button" class="close-button">&times;</button>
+      </div>
+      <nav>
+        <a href="./index.html" class="navigate"> Home </a>
+        <a href="about.html" class="navigate"> About </a>
+        <a href="./index.html#program" class="navigate"> Program </a>
+        <a href="#speakers-section" class="navigate"> Speakers </a>
+        <a href="./about.html#sponsor" class="navigate"> Sponsor </a>
+      </nav>
+    </div>
+    <header>
+      <div class="social-media">
+        <nav class="social-nav">
+          <a href="https://facebook.com/esther.udondian"
+            ><img src="./images/facebook-icon.png" alt="facebook"
+          /></a>
```

Comment:
- [ ] The facebook icon image is broken. It would be nicer to fix it to improve the UI design.

<img width="470" alt="image" src="https://user-images.githubusercontent.com/77190203/176998107-073ffbfa-6615-456e-9e58-d06e934b6ec8.png">